sim/mcuboot-sys/csupport/config-add-psa-crypto.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (c) 2023 Arm Limited
+ */
+
+#ifndef MCUBOOT_MBEDTLS_CONFIG_ADD_PSA_CRYPTO_H
+#define MCUBOOT_MBEDTLS_CONFIG_ADD_PSA_CRYPTO_H
+
+#include "mbedtls/build_info.h"
+
+/* Enable PSA Crypto Core without support for the permanent storage
+ * Don't define MBEDTLS_PSA_CRYPTO_STORAGE_C to make sure that support
+ * for permanent keys is not enabled, as it is not usually required during boot
+ */
+#define MBEDTLS_PSA_CRYPTO_C
+#define MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG
+
+#if defined(MCUBOOT_ENCRYPT_RSA) || defined(MCUBOOT_SIGN_RSA)
+    #define MBEDTLS_PK_C
+    #define MBEDTLS_ASN1_PARSE_C
+    #define MBEDTLS_CTR_DRBG_C
+    #define MBEDTLS_CIPHER_C
+    #define MBEDTLS_ENTROPY_C
+    #define MBEDTLS_PK_PARSE_C
+    #define MBEDTLS_PK_WRITE_C
+#endif /* defined(MCUBOOT_ENCRYPT_RSA) */
+
+#if defined(MCUBOOT_ENCRYPT_EC256) || defined(MCUBOOT_ENCRYPT_X25519)
+    #define MBEDTLS_CIPHER_MODE_CTR
+    #define MBEDTLS_PLATFORM_FREE_MACRO free
+    #define MBEDTLS_PLATFORM_CALLOC_MACRO calloc
+    #define MBEDTLS_AES_C
```

Comment:
`MBEDTLS_CIPHER_MODE_CTR` and `MBEDTLS_AES_C` are already defined in `config-ec.h` and `config-ed25519.h`. I believe these are duplications, but please correct me if I'm wrong.